chapter_appendix-mathematics-for-deep-learning/statistics.md

```diff
@@ -243,7 +243,7 @@ It is also referred to as the *type I error* or *false positive*. The $\alpha$,
 
 The *statistical power* (or *sensitivity*) measures the probability of reject the null hypothesis, $H_0$, when it should be rejected, i.e.,
 
-$$ \text{statistical power }= P(\text{reject } H_0  \mid H_0 \text{ is false} ).$$
+$$ \text{statistical power }= 1 - \beta = P(\text{ fail to reject } H_0  \mid H_0 \text{ is false} ).$$
```

Comment:
This is the definition of statistical power... Can you change it back?

Reply:
What I intend here is to make the style of this formula be consist with the one of the statistical significance. But I make a mistake here that I miss "1-" in front of P. The orignal formula is ok. 
Could I add the missing "1-" or change it back ?

Reply:
I see. Yes, let's add that " 1 - \beta = "!

Reply:
Ok!